src/box/lua/config/instance_config.lua

```diff
@@ -1357,6 +1357,50 @@ return schema.new('instance_config', schema.record({
             box_cfg = 'password_history_length',
         })),
     }),
+    metrics = schema.record({
```

Comment:
AFAIR, I usually leave comments, where we can't handle box.cfg options using box_cfg annotation. Please, look at other examples in the instance config schema and mark this option accordingly.

Reply:
Done.